### PR TITLE
fix(createImageRenderingActor): handle errors making new image scale

### LINKED ISSUE #544: Check for request of too large of an image

### PR DESCRIPTION
**Pull request by PaulHax**
Update scale drop-down to last successfully loaded scale after error loading user selected scale.

Good test case:
http://localhost:8082/?rotate=false&image=https://dandiarchive.s3.amazonaws.com/zarr/3d313fc2-0204-496d-bfa1-5c90951ee640
Then select scale 3 for "Out of memory error transferring to worker" or scale 2 for "to many network requests" error.

closes #544

### CODE REVIEW

File: src/Context/ImageActorContext.js [hunk 1]
```diff
@@ -2,9 +2,12 @@ class ImageActorContext {
   // MultiscaleSpatialImage to be visualized
   image = null
 
-  // The rendered image / label image scale
+  // The target image scale
   renderedScale = null
```
Comment (thewtex) [inline src/Context/ImageActorContext.js:6]: To be accurate with the naming here, should this be called `targetScale` and `loadedScale` -> `renderedScale` ?

Comment (PaulHax) [reply src/Context/ImageActorContext.js]: Good idea.  Changed.  It breaks the Bootstrap and Material GUIs though 🤦

Comment (thewtex) [reply src/Context/ImageActorContext.js]: Thanks -- could you please address Bootstrap and Material after merge?

Comment (PaulHax) [reply src/Context/ImageActorContext.js]: Will do.

**Review by thewtex — APPROVED** [review]

**Pull request merged**